### ISSUE TITLE
Add additional sections to development best practices

### ISSUE THREAD

**Issue by ddobrin** [networknt/light-doc#16]
Additions to be added in:
https://doc.networknt.com/development/best-practices/

To cover best practices on:
exception handling, performance, dependency injection, etc.

**Comment by stevehu** [comment]
Yes. That is the idea. Currently, there is only one section for swagger and openapi specification. Once we have the content, we can add them to the same page with different sections.
